.github/workflows/_e2e.yml

```diff
@@ -102,6 +105,17 @@ jobs:
           path: apps/explorer/playwright-report/
           retention-days: 30
 
+      - name: Run Dashboard e2e tests
+        if: inputs.isWalletDashboard || inputs.isRpc || inputs.isTypescriptSDK || github.ref_name == 'develop'
+        run: pnpm --filter wallet-dashboard playwright test
+
+      - uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4.4.3
+        if: always()
```

Comment:
```suggestion
        if: (!cancelled() && !failure())
```

Reply:
Mmmm no, just like the e2e test of wallet and explorer, these are meant to run even if the rest fails